### PR TITLE
VEN-503 | Single customer page: Fetch application data from the backend

### PR DESCRIPTION
**Pull request by TatuArvela**
## Description :sparkles:

* Added fetching application data for customers
* Switched all usages to the same `ApplicationDetails` component
* Due to backend changes, renamed harbor `maximumWidth` to `maxWidth`
* Made Table subComponent have same padding as CardBody (units(1.5))

## Issues :bug:

* Due to the Table subComponent change, the CustomerDetails in CustomerListComponent currently has more padding than before. To fix it, it needs the same change as ApplicationDetails.

### Closes :no_good_woman:

[VEN-503](https://helsinkisolutionoffice.atlassian.net/browse/VEN-503): Single customer page: Fetch application data from the backend

### Automated tests :gear:️

Updated tests for:
* ApplicationDetails
* ApplicationsCard

It would be great to improve the overall test coverage of the application.

### Manual testing :construction_worker_man:

Check:
* single customer page
* expanded rows on all applications page
* single application page

### CODE REVIEW

File: src/domain/individualCustomer/applicationsCard/ApplicationsCard.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+  applications: Application[];
+}
+
+const ApplicationsCard: React.SFC<ApplicationsCardProps> = ({
+  applications,
+}) => {
+  const { t } = useTranslation();
+
+  return (
+    <Card className={styles.applicationCard}>
```
Comment (Nizar-Rahme) [inline src/domain/individualCustomer/applicationsCard/ApplicationsCard.tsx:52]:
```suggestion
    <Card className={styles.applicationsCard}>
```

File: src/domain/individualCustomer/IndividualCustomerPageContainer.tsx [hunk 2]
```diff
@@ -13,10 +13,10 @@ import CardBody from '../../common/cardBody/CardBody';
 import BillsCard from './billsCard/BillsCard';
 import CustomerInfoCard from '../cards/customerInfoCard/CustomerInfoCard';
 import LoadingSpinner from '../../common/spinner/LoadingSpinner';
-import ApplicationCard from './applicationCard/ApplicationCard';
+import ApplicationCard from './applicationsCard/ApplicationsCard';
```
Comment (Nizar-Rahme) [inline src/domain/individualCustomer/IndividualCustomerPageContainer.tsx:16]: Here you might consider changing the name to `ApplicationsCard`.

File: src/domain/individualCustomer/applicationsCard/ApplicationsCard.tsx (hunk 1 above)
Comment (TatuArvela) [reply src/domain/individualCustomer/applicationsCard/ApplicationsCard.tsx]: Thanks! I made a commit fixing this

File: src/domain/individualCustomer/IndividualCustomerPageContainer.tsx (hunk 2 above)
Comment (TatuArvela) [reply src/domain/individualCustomer/IndividualCustomerPageContainer.tsx]: Thanks! I made a commit fixing this

**Pull request merged**